### ISSUE TITLE
Preferences

### ISSUE THREAD

**Issue by lskatz** [marbl/Krona#57]
I'm wondering if there is a way to have Krona preferences (font size, chart size, etc)?  Either when the HTML file is made (e.g., `ktImportText --font-size 16 --chart-size +1`) or through browser cookies.

**Comment by ondovb** [comment]
Hi Lee- I think the closest thing available is adding GET variables to the URL, e.g.:
http://marbl.github.io/Krona/examples/phymmbl.krona.html?font=15

If that works for you, the best way to see what's available is just using the "link" button in a chart. There is no chart size variable, unfortunately.

**Comment by lskatz** [comment]
Thanks Brian! It's something I can do short term. If I can figure it out (big if), would you accept a pull request for cookies or command line parameters?

**Comment by ondovb** [comment]
Definitely! The code is not well documented but I can try to walk you through anything that doesn't make sense.

**Comment by khyox** [comment]
Hi! I think it would be very nice to have those preferences (or any other) also available through the Krona XML specification. Maybe I can help with that part. I am currently working in a python interface to Krona by the 2.0 XML specification so I would see very interesting to have all the new functionality available this way. So, Brian, if you find this also useful, I would be glad to contribute!

**Comment by ondovb** [comment]
Sure, anything through the command line would pretty much have to alter the XML spec. It should stay backwards compatible as long as we're just adding things though.

**Comment by lskatz** [comment]
I defer to @khyox if he or she knows how to code this! Or was your offer for only the documentation?

**Comment by ondovb** [comment]
I can answer any questions about the code or architecture as needed. @khyox do you want to propose the xml changes here, and if it works for @lskatz move forward with a PR?

**Comment by khyox** [comment]
@ondovb, @lskatz, all right, so I understand that the XML spec should be changed as a prerequisite for including those options in the command line, is that right? In that case, yes, I think we had better clarify here the xml changes that include those options (and maybe some other) and then try to go ahead with a PR.

**Comment by lskatz** [comment]
That can work.  From what I can tell it might be easiest to work through `ktImportText` since I mostly want to embed the defaults into the file.  All I can tell so far is that I would want to mess with `writeTree()`

**Comment by ondovb** [comment]
That's right. More specifically, `dataHeader()`, called from `writeTree()`. You can see there is already a `collapse` option that functions more or less how we want, but is not exposed to the command line.
